### PR TITLE
Fixed Onboarding page as requested #Fixes 5740

### LINKED ISSUE #5740: Make the onboarding full screen on mobile.
# Current Behavior

We display a mobile onboarding popup, but it doesn't render well.

CleanShot 2024-06-05 at 10 29 03

# Desired Behavior

Make the content full-screen on mobile:

image

# Figma

https://www.figma.com/design/xt8O9mFeLl46C5InWwoMrN/Twenty?node-id=32468-360264&t=XbbQ79dyfHCjEchG-11

### PR DESCRIPTION
**Pull request by sagarhedaoo**
Closes #5740

Screenshot 2024-06-06 at 2 26 41 AM
Screenshot 2024-06-06 at 2 26 54 AM

Fixed the onboarding page as shown in figma design. if any further changes are required, I am happy to do that!

### CODE REVIEW

File: packages/twenty-front/src/modules/ui/layout/modal/components/ModalLayout.tsx [hunk 1]
```diff
@@ -42,6 +42,18 @@ const StyledModalDiv = styled(motion.div)<{
         return 'auto';
     }
   }};
+
+  @media (max-width: 768px) {
```
Comment (FelixMalfait) [inline packages/twenty-front/src/modules/ui/layout/modal/components/ModalLayout.tsx:46]: @media (max-width: ${MOBILE_VIEWPORT}px) {

**Comment by FelixMalfait** [comment]
Thanks for the contribution. This doesn't look great for existing modales.

cc @Bonapara 

Screenshot 2024-06-06 at 15 15 57

Screenshot 2024-06-06 at 15 14 49


We probably want to differentiate when the modal can be closed VS when it cannot.
When it can be closed do we want a bottom sheet? or a close button @Bonapara?

**Comment by FelixMalfait** [comment]
Also It's not properly centered on all screen sizes:

Screenshot 2024-06-06 at 15 20 38

**Comment by FelixMalfait** [comment]
Screenshot 2024-06-06 at 15 22 01

**Comment by Bonapara** [comment]
> Thanks for the contribution. This doesn't look great for existing modales.
> 
> cc @Bonapara
> 
> Screenshot 2024-06-06 at 15 14 49
> We probably want to differentiate when the modal can be closed VS when it cannot. When it can be closed do we want a bottom sheet? or a close button @Bonapara?

Could we keep the confirmation modal as it is today and make this PR affect only the onboarding flow?

Desired behavior for modal (like today):
image

**Comment by Bonapara** [comment]
> Screenshot 2024-06-06 at 15 22 01

I think we can keep the desktop design on the tablet since the screen is already large enough.

**Comment by Bonapara** [comment]
Hi @sagarhedaoo, thanks for taking this issue!

We should introduce the concept of "variants" for the modal to allow the onboarding to look different from the alerts inside the app.

Here are their specs:

  | Primary | Secondary
-- | -- | --
Laptop | Background #1B1B1B opacity 80% | Background #1B1B1B opacity 40%
Mobile | Fullscreen | Background #1B1B1B opacity 40%
Ability to close by clicking outside or by pressing  ⎋ (esc) | Option (Can be activated or not) | Option (Can be activated or not)

How does it sound?

**Comment by sagarhedaoo** [comment]
@Bonapara 
Variants can be interesting to try! Do you want me to implement variants for the modal?

**Comment by FelixMalfait** [comment]
@sagarhedaoo should we close this? otherwise would be great if you can raise a clean fix with the above feedback! Thanks a lot

**Comment by lucasbordeau** [comment]
@ehconitin I'm closing this PR because you're handling this in https://github.com/twentyhq/twenty/pull/6386

**Pull request closed without merging**